packages/ui/src/app/pages/Election/BlacklistedAccounts/BlacklistedAccount.tsx

```diff
@@ -0,0 +1,56 @@
+import { Identicon } from '@polkadot/react-identicon'
+import React from 'react'
+import styled from 'styled-components'
+
+import { AccountCopyAddress, AccountInfoWrap, AccountPhoto, PhotoWrapper } from '@/accounts/components/AccountInfo'
+import { useBalance } from '@/accounts/hooks/useBalance'
+import { AccountRow, BalanceInfoInRow, InfoTitle, InfoValue } from '@/common/components/Modal'
+import { TokenValue } from '@/common/components/typography'
+import { Colors } from '@/common/constants'
+import { shortenAddress } from '@/common/model/formatters'
+
+interface Prop {
+  address: string
+}
+
+export const BlacklistedAccount = ({ address }: Prop) => {
+  const balance = useBalance(address)
```

Comment:
For most users `useBalance` will call `derive.balances.all`. RPC node queries are not cached and they should be reduced as much as possible. Since `useAddresses` is called on this page, this component shouldn't call `useBalance`. Instead:
```suggestion
export const BlacklistedAccount = ({ address, balance }: Prop) => {
```